test/test_extralite.rb

```diff
@@ -6,4 +6,28 @@ class ExtraliteTest < MiniTest::Test
   def test_sqlite3_version
     assert_match /^3\.\d+\.\d+$/, Extralite.sqlite3_version
   end
+
+  def test_status
+    db = Extralite::Database.new(':memory:')
+    db.query('create table if not exists t (x,y,z)')
+    db.query('insert into t values (1, 2, 3)')
+
+    begin
+      a = Extralite::runtime_status(Extralite::SQLITE_STATUS_MEMORY_USED, false)
+      b = Extralite::runtime_status(Extralite::SQLITE_STATUS_MEMORY_USED)
+      c = Extralite::runtime_status(Extralite::SQLITE_STATUS_MEMORY_USED, true)
+      d = Extralite::runtime_status(Extralite::SQLITE_STATUS_MEMORY_USED, true)
+
+      assert_operator 0, :<, a[0]
+      assert_operator a[0], :<=, a[1]
+      
+      assert_equal a, b
+      assert_equal a, c
+
+      assert_equal a[0], d[0]
```

Comment:
why I removed those 2 lines from my PR is because they actually were racy for no reason - the results in some CIs were not consistent.

Reply:
All checks for this PR have passed: https://github.com/digital-fabric/extralite/actions/runs/3976516163.

If you're still seeing failures when running the tests, try to isolate it and open an issue. Thanks!